lib/validation.js

```diff
@@ -105,9 +106,21 @@ function isValidLogger (logger) {
   return result
 }
 
-function schemaCompiler (schema) {
-  return this.ajv.compile(schema)
+function buildSchemaCompiler () {
+  // This instance of Ajv is private
+  // it should not be customized or used
+  const ajv = new Ajv({
+    coerceTypes: true,
+    useDefaults: true,
+    removeAdditional: true
+  })
+
+  return schemaCompiler
```

Comment:
Why not `ajv.compile.bind(ajv)`?

Reply:
Old habit from Node 6. I would prefer to leave it as it is.